kodilla-stream/src/main/java/com/kodilla/stream/homework/TaskManager.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.stream.homework;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskManager {
+    public static void main(String[] args) {
+        LocalDate currentDate = LocalDate.now();
+        List<LocalDate> deadlineTask = TaskRepository.getTasks()
+                .stream()
+                .filter(dl -> dl.getDeadline().isAfter(currentDate))
+                .map(TaskManager :: getTaskDate)
+                .collect(Collectors.toList());
+
+        System.out.println("Lista dat wszystkich zadań, których data jeszcze nie upłynęła: \n" + deadlineTask);
```

Comment:
to nie zadziała. Musisz mieć metodę, która wypisze wszystkie daty z tej listy (możesz znów zrobić stream w tej metodzie).

Reply:
dziwne, bo zadziałało i z tym moim przykładem TaskManager :: getTaskDate i sout